appveyor.yml

```diff
@@ -15,6 +16,13 @@ skip_tags: true
 
 clone_folder: c:\projects\habitat
 
+environment:
+  matrix:
+    - hab_build_action: build
+      hab_components: hab
```

Comment:
Yeah I'd add `hab` to the list of components to be tested. Do you think there's much value in a matrix just ensuring the component builds? The cargo test task will build the software in addition to running the test suite. It's true that we could be using `cfg(test)` for conditional compilation outside of just marking modules as test modules, but we don't really do that.

Whatever you feel is most appropriate just make sure we run the test suite on all components ;)